sound/soc/intel/boards/bxt_da7219_max98357a.c

```diff
@@ -643,9 +646,6 @@ static int bxt_card_late_probe(struct snd_soc_card *card)
 		i++;
 	}
 
```

Comment:
And the above hdac-specific setting could be put to hda_dsp_common.h. I didn't put it in my original series as I wanted to keep the series as compact as possible, but would reduce overlapping code. Can be done in a separate PR as well.